sentry_sdk/client.py

```diff
@@ -555,11 +578,18 @@ def capture_event(
         :param scope: An optional scope to use for determining whether this event
             should be captured.
 
+        :param scope_kwargs: For supported `**scope_kwargs` see
+            :py:meth:`sentry_sdk.Scope.update_from_kwargs`.
```

Comment:
This is now not entirely true since we've added `top_scope` to possible `scope_kwargs`, but I'm also not sure if we should update the docstring for `update_from_kwargs` with that since `top_scope` is only used internally.

On the other hand I'm thinking it might make sense to take `top_scope` out of the kwargs and make it its own keyword arg, not just in `capture_event` but also the other `capture_*` functions here. Doing that would clearly separate it from `scope_kwargs` which have a slightly different purpose and the docstring would still be correct. What do you think?